client/styles/components/_hints.scss

```diff
@@ -0,0 +1,182 @@
+.CodeMirror-hints {
+  position: absolute;
+  z-index: 10;
+  overflow: hidden;
+  list-style: none;
+
+  margin: 0;
+  padding: 0;
+
+  -webkit-box-shadow: 0 0 18px 0 rgba(0, 0, 0, 0.16);
+  -moz-box-shadow: 0 0 18px 0 rgba(0, 0, 0, 0.16);
+  box-shadow: 0 0 18px 0 rgba(0, 0, 0, 0.16);
```

Comment:
I notice in this file you're using `px` sometimes, `em` sometimes, and `rem` sometimes. I prefer to use `rem`, but because I usually think in pixels, I'll use this trick for SCSS: `#{50 / $base-font-size}rem` where 50 is the measurement in pixels.